### PR TITLE
Mark.viterna/api 5959 add revision on pdf

### PR DESCRIPTION
**Pull request by mviterna**
We are troubleshooting some issues where paper trail is breaking a new endpoint field. We have some data to troubleshoot and now need to roll the code back
## Original issue(s)
https://vajira.max.gov/browse/API-5959

## Things to know about this PR

Got JSON response needed. Now I'm able to reproduce locally

### CODE REVIEW

File: spec/support/schemas_camelized/va_forms/forms.json [hunk 1]
```diff
@@ -54,10 +54,7 @@
               },
               "sha256": {
                 "type": "string"
-              },
-              "lastSha256Change": {
-                "type": ["string", "null"]
-              },
+              }
```
Comment (ericboehs) [inline spec/support/schemas_camelized/va_forms/forms.json:57]:
```suggestion
              },
```

This json is invalid.

Comment (mviterna) [reply spec/support/schemas_camelized/va_forms/forms.json]: FIxed. Yeah, this one is tricky. I can't reproduce it locally. But I found some records in staging that are blowing up, so I'm going to copy them down locally and try to troubleshoot

**Comment by ericboehs** [comment]
Do these need to be removed as well?

https://github.com/department-of-veterans-affairs/vets-api/blob/fe456094502ef50bc20748a27bb7f305288563b4/modules/va_forms/config/locales/en.yml#L24
https://github.com/department-of-veterans-affairs/vets-api/blob/fe456094502ef50bc20748a27bb7f305288563b4/modules/va_forms/app/swagger/va_forms/forms/form_swagger.rb#L64
https://github.com/department-of-veterans-affairs/vets-api/blob/fe456094502ef50bc20748a27bb7f305288563b4/spec/support/schemas/va_forms/forms.json#L35

**Comment by mviterna** [comment]
> Do these need to be removed as well?
> 
> https://github.com/department-of-veterans-affairs/vets-api/blob/fe456094502ef50bc20748a27bb7f305288563b4/modules/va_forms/config/locales/en.yml#L24
> 
> 
> https://github.com/department-of-veterans-affairs/vets-api/blob/fe456094502ef50bc20748a27bb7f305288563b4/modules/va_forms/app/swagger/va_forms/forms/form_swagger.rb#L64-L65
> 
> 
> https://github.com/department-of-veterans-affairs/vets-api/blob/fe456094502ef50bc20748a27bb7f305288563b4/spec/support/schemas/va_forms/forms.json#L35

Well, I can. I found the bug and I'm going to get it in the next deploy tomorrow, so the swagger stuff will need to be in there eventually. Whatever you want me to do

**Comment by mviterna** [comment]
@ericboehs  This is the fix :       object.versions.last.try(:created_at&.try(strftime('%Y-%m-%d'))) if @item.respond_to?(:versions)

Is it worth trying to get it in this deploy?

**Comment by ericboehs** [comment]
Is there a test that can be written that will verify this change is behaving as it should?

**Review by ericboehs — APPROVED** [review]

**Review by gpbowman-git — APPROVED** [review]

**Pull request merged**